tests/components/homeassistant/triggers/test_time.py

```diff
@@ -170,22 +166,19 @@ async def test_if_fires_using_multiple_at(hass, calls):
                 }
             },
         )
+        await hass.async_block_till_done()
 
     now = dt_util.utcnow()
```

Comment:
I think this is unused now.  Not going to hold up the PR though since we need to get the CI working again

Reply:
Agreed, looks unused.